app/contact/page.tsx

```diff
@@ -1,3 +1,135 @@
-export default function Contact() {
-  return null;
+import React from "react";
+import Image from "next/image";
+import discordImg from "../../public/img/discord.png";
+import instagramImg from "../../public/img/insta-color.png";
+import linkedinImg from "../../public/img/linkedin.png";
+
+function Contact() {
+  return (
+    <div className="bg-[#1e1e1e] text-white font-shareTech min-h-screen flex flex-col justify-center items-center pt-[8rem]">
```

Comment:
```suggestion
    <div className="bg-[#1e1e1e] text-white min-h-screen flex flex-col justify-center items-center pt-[8rem]">
```
Due to the work done in #11, removing the explicit font set using tailwind might be better.